scripts/Muon/GUI/ElementalAnalysis2/context/ea_group_context.py

```diff
@@ -10,8 +10,8 @@
 from Muon.GUI.ElementalAnalysis2.context.context import REBINNED_VARIABLE_WS_SUFFIX, REBINNED_FIXED_WS_SUFFIX
 from Muon.GUI.ElementalAnalysis2.auto_widget.ea_auto_tab_model import PEAKS_WS_SUFFIX, MATCH_GROUP_WS_SUFFIX
 
-INVALID_STRINGS_FOR_GROUP_NAMES = ["_EA_Rebinned_Fixed", "_EA_Rebinned_Variable", "_peaks", "_refitted_peaks",
-                                   "_with_errors", "_matches"]
+INVALID_STRINGS_FOR_GROUP_NAMES = ["_EA_Rebinned_Fixed", "_EA_Rebinned_Variable", "_EA_peaks", "_EA_refitted_peaks",
```

Comment:
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/ElementalAnalysis2/context/context.py#L14 and do we have similar strings for others too?

Reply:
I am not sure what you mean but do you want me to use the variable names in list instead of string

Reply:
you should use the variable as in the link. So if it changes in the future it is changed everywhere, instead of potentially missing this one and breaking the code